### ISSUE TITLE
MobileElement.findElement doesn't work for iOS IosUIAutomation + AccessibilityId

### ISSUE THREAD

**Issue by Sergey-Kim** [appium/appium#4512]
Hi

I'm trying to do the following:
MobileElement element = driver.findElement(MobileBy.IosUIAutomation(...));
element.findElement(MobileBy.AccessibilityId(...));

and the element.findElement fails for me. With element cannot be located... message
If I change first find strategy to xpath, everything seems to be fine.

I work with Appium 1.3.4, Java bindings 2.1.0, iOS Emulator 8.1

**Comment by Sergey-Kim** [comment]
hello?

**Comment by sebv** [comment]
Hi, 

Sorry for the delay,

Could you print the page source and provide the log, could be a visibility issue, but there is not enough information here? (Ideally it would be good to have reproducible cases so we can debug.)

**Comment by sebv** [comment]
Closing, no activity, may be reopened if needed.
